### ISSUE TITLE
Problem after upgrading to 3.1.0

### ISSUE THREAD

**Issue by uLan08** [langpavel/node-pg-async#104]
Using this connection uri `postgres://postgres@localhost:5432/db` gives out error `rolename does not exist`. It was working fine before updating. Is there something that I may have missed?

**Comment by langpavel** [comment]
Yes.. connection string is now passed in a little different way:
https://node-postgres.com/features/connecting#connection-uri

**Comment by uLan08** [comment]
I see. Thanks for this.
